### PR TITLE
refactor(TelegramTrigger Node, ShopifyTrigger Node): Standardize node triggers actions

### PR DESCRIPTION
**Pull request by OlegIvaniv**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: packages/editor-ui/src/stores/nodeCreator.ts [hunk 1]
```diff
@@ -166,9 +166,11 @@ function recommendedCategory(
 				i18n.baseText('nodeCreator.actionsCategory.recommended'),
 			),
 			actionKey: categoryItem.value as string,
-			displayName: i18n.baseText('nodeCreator.actionsCategory.onEvent', {
-				interpolate: { event: startCase(categoryItem.name) },
-			}),
+			displayName:
+				categoryItem.action ??
```
Comment (ivov) [inline packages/editor-ui/src/stores/nodeCreator.ts:170]: Are we okay with no i18n in this case?

**Review by ivov — APPROVED** [review]
LGTM. PR title should prolly use `refactor` and omit node names, already mentioned in scope.

File: packages/editor-ui/src/stores/nodeCreator.ts (hunk 1 above)
Comment (OlegIvaniv) [reply packages/editor-ui/src/stores/nodeCreator.ts:170]: This override comes from the Node definition for which we're not using i18n

Comment (ivov) [reply packages/editor-ui/src/stores/nodeCreator.ts:170]: Exactly! In almost all cases this string is translatable but for overrides it's not, so not ideal for someone localizing. Just to confirm this is okay, minor thing.

**Comment by janober** [comment]
Got released with `n8n@0.211.0`

**Pull request merged**